File: src/main/java/xyz/chunkstories/api/plugin/commands/ServerConsole.java
//
// This file is a part of the Chunk Stories API codebase
// Check out README.md for more information
// Website: http://chunkstories.xyz
//

package xyz.chunkstories.api.plugin.commands;

import xyz.chunkstories.api.server.Server;

public interface ServerConsole extends CommandEmitter {
	public Server getServer();
}

File: src/main/java/xyz/chunkstories/api/plugin/commands/ServerConsole.kt
//
// This file is a part of the Chunk Stories API codebase
// Check out README.md for more information
// Website: http://chunkstories.xyz
//

package xyz.chunkstories.api.plugin.commands

import xyz.chunkstories.api.server.Host

/** Represents the console of a standalone server */
interface ServerConsole : CommandEmitter {
    val server: Host
}
